commands/keplr.js

```diff
@@ -287,22 +324,25 @@ const keplr = {
 
     return true;
   },
-
-  async addNewTokensFound() {
-    module.exports.switchToKeplrIfNotActive();
-    await module.exports.goToHome();
+  async addNewTokensFound(switchScreens = true) {
+    if (switchScreens) {
+      module.exports.switchToKeplrIfNotActive();
+      await module.exports.goToHome();
+    }
     await playwright.waitAndClickByText(homePageElements.newTokensFound);
-    await playwright.waitAndClick(
+    await playwright.waitAndClickWithRetry(
       homePageElements.selectAllTokensCheck,
-      playwright.keplrWindow(),
       { number: -1, force: true },
     );
     await playwright.waitAndClickByText(
       homePageElements.addChainsButton,
       playwright.keplrWindow(),
       true,
     );
-    await playwright.switchToCypressWindow();
+
+    if (switchScreens) {
+      await playwright.switchToCypressWindow();
+    }
```

Comment:
i think we should always be switching screens regardless. 
currently we're building our flows with the mindset that every interaction/command will switch to the keplr window, perform its interaction, and switch back to the cypress window.
if we start performing interactions that deviate from that, it could break the flow of subsequent interactions

Reply:
I needed to reuse this implementation in a flow I was implementing without the screen-switching part. All the tests were passed and now flow was broken.

Reply:
what flow was this? i dont see it being used anywhere
the `should get the accurate values for the tokens in the wallet` spec uses `getTokenAmount` twice in succession and it doesn't have any issues.

Reply:
Please see the `getWalletAddress` function